File: responses/A5QMTE-D2P/sum_of_integers_in_a_string.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

# Misread the problem statement. Thought it was looking for index, not count...

def solution(A):
    # write your code in Python
    deduped = set(A)
    # print(deduped)
    biggest = max(deduped)
    # print(biggest)
    size_a = len(A)
    i = 0
    for i in range(size_a):
        if A[i] < size_a and A[i] == biggest:
            return i
        elif i == size_a:
            return 0
        i += 1
    pass

File: responses/VWDVV4-W6X/sum_of_integers_in_a_string.js
// you can write to stdout for debugging purposes, e.g.
// console.log('this is a debug message');

function solution(A) {
    // write your code in JavaScript
    if (A.length == 0) {
        return 0;
    }
    var max = 0;
    var map = {};
    for (var i = 0; i < A.length; i++) {
        var val = A[i];
        if (val > max) {
            max = val;
        }
        if (map[val]) {
            map[val] += 1;   
        }
        else {
            map[val] = 1;   
        }
    }
    return map[val];
}